package.json

```diff
@@ -305,11 +315,13 @@
         "vscode:prepublish": "npm run recompile",
         "bundleDeps": "node ./build-scripts/bundleDeps.js",
         "clean": "node ./build-scripts/clean.js out",
+        "createWelcomePage": "node ./build-scripts/createWelcomePage.js",
         "reset": "node ./build-scripts/clean.js out node_modules && npm install",
         "copyNonCodeFiles": "node ./build-scripts/copyNonCodeFiles.js",
-        "compile": "tsc -p ./ && npm run lint && npm run bundleDeps && npm run copyNonCodeFiles",
+        "buildScripts": "npm run bundleDeps && npm run copyNonCodeFiles && npm run createWelcomePage",
+        "compile": "tsc -p ./ && npm run lint && npm run buildScripts",
         "recompile": "npm run clean && npm run compile",
-        "watch": "npm run copyNonCodeFiles && tsc -watch -p ./",
+        "watch": "npm run buildScripts && tsc -watch -p ./",
```

Comment:
This now runs `npm run bundleDeps`. Was this change intentional, and is there a negative impact in watch iterations?

Reply:
See: https://github.com/aws/aws-toolkit-vscode/pull/619#discussion_r294525947